.github/labels.yml

```diff
@@ -67,6 +67,9 @@ labels:
 - name: 'dependencies'
```

Comment:
Could this be changed to `dependencies/python` for clarity?

Reply:
i think dependabot always adds the dependencies label regardless of what dependency it is.